index.html

```diff
@@ -5,36 +5,46 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Mixtape</title>
+    <title>Mixtape Setup</title>
+
+    <link href="assets/css/bootstrap.min.css" rel="stylesheet">
```

Comment:
@edumoran looks to me that we are including the bootstrap css two times. Maybe we should remove one reference.